Python/Searches/binary_search.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+def main():
+    array = []
+    print("Enter numbers to add in array and -9999 to stop")
+    while True:
```

Comment:
Better ask the user about number of inputs and input the value for that many iterations instead of a hard-coded default.

This coding style can fail very easily. :smiley:

Reply:
OK!